src/Altinn.App.Core/Interface/IProcess.cs

```diff
@@ -19,7 +19,7 @@ public interface IProcess
         /// <param name="instance">the instance</param>
         /// <param name="events">process events</param>
         /// <returns></returns>
-        public Task DispatchProcessEventsToStorage(Instance instance, List<InstanceEvent> events);
+        public Task DispatchProcessEventsToStorage(Instance instance, List<InstanceEvent>? events);
```

Comment:
Given the name and description of this interface, does the DispatchPRocessEventsToStorage method belong here? Should this be a IProcessReader and nothing more?